src/plugins/analysis/hash/test/test_plugin_hash.py

```diff
@@ -1,8 +1,21 @@
+import os
+
 from test.unit.analysis.analysis_plugin_test_class import AnalysisPluginTest
 
+from common_helper_files import get_dir_of_file
 from ..code.hash import AnalysisPlugin
 from objects.file import FileObject
 
+TEST_DATA_DIR = os.path.join(get_dir_of_file(__file__), 'data')
```

Comment:
Could you move the MockObject to some common test helper module?
Maybe there even already exists one that you can use.